extensions/positron-python/python_files/posit/positron/tests/test_data_explorer.py

```diff
@@ -2551,8 +2551,8 @@ def test_pandas_profile_summary_stats(dxf: DataExplorerFixture):
             # decimal
             "f12": [
                 Decimal("1.5"),
-                Decimal("2"),
-                Decimal("2"),
+                Decimal(2),
+                Decimal(2),
```

Comment:
just out of curiosity-- why does this not have to be a string but others are?

Reply:
Great question!! https://docs.astral.sh/ruff/rules/verbose-decimal-constructor/#why-is-this-bad

>Note that this rule does not flag quoted float literals such as `Decimal("0.1")`, which will produce a more precise `Decimal` value than the unquoted `Decimal(0.1)`.